docs/development/virtual_environments.rst

```diff
@@ -195,18 +203,15 @@ Add your user to the kvm group to give it permission to run KVM:
    sudo modprobe kvm
    sudo modprobe kvm_intel
 
-
-Validate libvirt config
-^^^^^^^^^^^^^^^^^^^^^^^
-
-Log out, then log in again. Verify that libvirt is installed and KVM is
-available:
+Log out, then log in again. Verify that libvirt is installed:
 
 .. code:: sh
 
-   libvirtd --version
-   kvm-ok
+   sudo libvirtd --version
```

Comment:
Using `sudo` sounds good to me. Not sure why that's needed. We really shouldn't be if the user is in the `libvirt` group, but this should be fine for now.

Reply:
Agree that we should remove `sudo`.

Reply:
Ah, so my issue is that there doesn't seem to be a reason why we need `sudo` but currently that is the case. Even if your local user is part of the `libvirt` and `kvm` groups, this is needed.

Reply:
There sholdn't be. You have to log out and login *completely*. Not just the terminal session. If you still need `sudo`, something went wrong.

Reply:
Oh man. I think you were right. I just had a live `sudo` session so I didn't hit that same error, but I hit it now.

Reply:
Looks like it's needed for checking libvirtd on Debian, but `sudo` isn't a requirement for regular use. If virt-manager and the cli tools are good, then we good?